src/Test/contconic.jl

```diff
@@ -496,11 +496,12 @@ function soc3test(model::MOI.ModelLike, config::TestConfig)
 
         MOI.optimize!(model)
 
-        @test MOI.get(model, MOI.TerminationStatus()) == MOI.INFEASIBLE
+        @test MOI.get(model, MOI.TerminationStatus()) in (MOI.INFEASIBLE, 
+                                                          MOI.INFEASIBLE_OR_UNBOUNDED)
```

Comment:
@odow Why do we need to support `INFEASIBLE_OR_UNBOUNDED`? We can set the following parameters so that Gurobi doesn't return this status:
https://www.gurobi.com/documentation/8.0/refman/infunbdinfo.html
https://www.gurobi.com/documentation/8.0/refman/dualreductions.html#parameter:DualReductions

Reply:
Oooo. I didn't realize there was this parameter.

Reply:
I've updated Gurobi to use the parameter, and reverted the change here.